tasks.py

```diff
@@ -339,12 +358,19 @@ def regenerate_legacy(c, swagger_name=None, debug=False):
         regenerate_samples(c, debug)
         regenerate_with_python3_operation_files(c, debug)
 
+@task
```

Comment:
Seeing all of the extra code we have to generate, and seeing that we want to combine operation groups for `--version-tolerant`, I think we can change the default to `combine-operation-groups` to be `true` if `--version-tolerant` is passed in. That way, you won't need to add all of this additional generation, and we should just see changes in the vesrion tolerant files. what do you think? cc @lmazuel

Reply:
`combine-operation-groups` can be used with both `version-tolerant` or `track2`. So it is more like individual function. In my opinion, it is better to test it individually and it will be easier to maintain in the future.

Reply:
I have a few reasons why it would be better to combine them
1. We're not trying to have this be true for `track2`, because even though the operation files are private, the CLI team has some code that depends on the full path to operations (they use this to mock), so making this true by default for track 2 will break them. 
2. This is just one of the flags that we're going to default to `True` for `--version-tolerant`, and is of similar weight to all of the flags [here](https://github.com/Azure/autorest.python/blob/autorestv3/autorest/codegen/__init__.py#L280), so I don't think it's worth it to add this much more test code
3. Our generation and testing for this CI is already pushing an hour, which we currently have the timeout for. I don't think it's worth it to add this much testing infrastructure, which will be harder to maintain in the future

Reply:
got it. I have changed the code : https://github.com/Azure/autorest.python/blob/202564490721ff668ae556b4931b0585390ed372/autorest/codegen/__init__.py#L288

`combine-operation-groups` will be test together with `version-tolerant`